src/modules/packages/packages.c

```diff
@@ -400,6 +406,7 @@ void ffPrintPackagesHelpFormat(void)
         "Number of winget packages",
         "Number of opkg packages",
         "Number of am packages",
+        "Number of spells",
```

Comment:
spells?

Reply:
This is as per official [Glossary](https://sourcemage.org/Glossary).

Reply:
The package manager name should be used because it's more intuitive, or brew will be called as formulae too

Reply:
Yeah, makes sense. Corrected.